src/System.Private.Reflection.Core/src/System/Reflection/Runtime/General/Dispensers.Ecma.cs

```diff
@@ -86,6 +87,7 @@ public override int GetHashCode()
             }
 
             public MetadataReader Reader { get; }
+            public string AssemblyPath { get; }
```

Comment:
I'm not super fond of putting this here but wasn't sure if closuring it instead for the creator function was a better idea.